tests/utils.go

```diff
@@ -4298,20 +4303,37 @@ func VolumeExpansionAllowed(sc string) bool {
 		*storageClass.AllowVolumeExpansion
 }
 
-func HasBindingModeWaitForFirstConsumer() bool {
+func IsStorageClassBindingModeWaitForFirstConsumer(sc string) bool {
 	virtClient, err := kubecli.GetKubevirtClient()
 	Expect(err).ToNot(HaveOccurred())
-	sc, exists := GetRWOFileSystemStorageClass()
-	if !exists {
-		return false
-	}
 	storageClass, err := virtClient.StorageV1().StorageClasses().Get(context.Background(), sc, metav1.GetOptions{})
-	Expect(err).ToNot(HaveOccurred())
 	if err != nil {
 		return false
 	}
 	return storageClass.VolumeBindingMode != nil &&
-		*storageClass.VolumeBindingMode == storagev1.VolumeBindingWaitForFirstConsumer
+		*storageClass.VolumeBindingMode == wffc
+}
+
+func GetStorageClassWithBindingModeWaitForFirstConsumer() (string, bool) {
```

Comment:
Don't think this is needed, the function generates a more or less random storage class anyway since there is no order, and if you have multiple WFFC storage classes things can get weird.